src/client/MemberItem.tsx

```diff
@@ -23,7 +23,7 @@ const useStyles = makeStyles({
   },
 });
 
-const MemberItem = ({ member, trx = {}, onVolumeClick }) => {
+const MemberItem = ({ member, trx, onVolumeClick }) => {
```

Comment:
It looks like this "fix" is not entirely correct yet as I am getting an error on trx.isRecving when I connect a second client.

Reply:
ok, my last commit fixes the 'issue' I had.
I modified the trx state handling to trigger also on ADD_MEMBER and costruct an initialTrxState that contains all the properties set to a default.